### PR TITLE
Updates glab from 1.22.0 to 1.24.1; fixes #2 

### LINKED ISSUE #2: Update from 1.22.0 to 1.24.1
The [GitLab repository CLI](https://gitlab.com/gitlab-org/cli) (also known as `glab`) has had a few new [releases](https://gitlab.com/gitlab-org/cli/-/releases) and by now is at [v1.24.1](https://gitlab.com/gitlab-org/cli/-/releases/v1.24.1) which contains support for [snippets](https://docs.gitlab.com/ee/user/snippets.html) (the [GitLab](https://en.wikipedia.org/wiki/GitLab) equivalent of [GitHub](https://en.wikipedia.org/wiki/GitHub) [gist](https://en.wikipedia.org/wiki/GitHub#Gist)s). 

So it is time to update for which I will post a Pull Request soon.

### PR DESCRIPTION
**Pull request by jpluimers**
Updates [GitLab CLI](https://gitlab.com/gitlab-org/cli) (glab) from [1.22.0](https://gitlab.com/gitlab-org/cli/-/releases/v1.22.0) to [1.24.1](https://gitlab.com/gitlab-org/cli/-/releases/v1.24.1); fixes https://github.com/corbob/ChocoPackages/issues/2

### CODE REVIEW

File: packages/glab/glab.nuspec [hunk 1]
```diff
@@ -26,4 +26,4 @@
       <dependency id="glab.portable" />
```
Comment (corbob) [inline packages/glab/glab.nuspec:26]:
```suggestion
      <dependency id="glab.portable" version="[1.24.1]" />
```

I saw recently a suggestion to be explicit with the dependency version numbers to ensure the proper packages are being installed.

File: packages/glab.portable/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+  url64bit      = $url64
+  checksum      = '85e9b4ee96337ebe48b040a552b2cb7327be86d235ff6e2c9d555be32c9fe56c'
   checksumType  = 'sha256'
-  checksum64    = '1b07931cbd9547fdf33bc5c636a5e70bc01d7ef68768cff13f9655623ce411d8'
+  checksum64    = '6b553a556ba7011a0c9fcf8a5f947ab60c6aabfaada47391ea7ba5703b08392b'
   checksumType64= 'sha256'
 }
-Install-ChocolateyZipPackage @packageArgs # https://docs.chocolatey.org/en-us/create/functions/install-chocolateyzippackage
\ No newline at end of file
+Install-ChocolateyZipPackage @packageArgs # https://docs.chocolatey.org/en-us/create/functions/install-chocolateyzippackage
```
Comment (corbob) [inline packages/glab.portable/tools/chocolateyinstall.ps1:16]:
```suggestion
Install-ChocolateyZipPackage @packageArgs # https://docs.chocolatey.org/en-us/create/functions/install-chocolateyzippackage

# Remove any zip files from previous installs
Remove-Item $toolsDir/*.zip
```

The `Install-ChocolateyZipPackage` doesn't remove the zip files, so they could be left behind between versions. This just makes sure the 1.22.0 files are removed.

**Review by corbob — APPROVED** [review]
Overall this looks good to me @jpluimers, thanks for getting this put together.

There are a few changes I'd like to add that are not actually from parts you've changed. I've  added the suggestions to this PR, and will commit them as well.

Along with removing the Verification.txt file as it's no longer necessary since the files aren't being embedded in the package anymore.

**Comment by corbob** [comment]
Thanks a lot for getting this updated @jpluimers. I've packed it and pushed it up to the Community Repository: https://community.chocolatey.org/packages/glab/1.24.1 and https://community.chocolatey.org/packages/glab.portable/1.24.1

**Comment by cforce** [comment]
curious about namings - gitlab-cli vs glab - intended?

**Pull request merged**